frontend/coprs_frontend/coprs/views/groups_ns/groups_general.py

```diff
@@ -41,12 +42,12 @@ def activate_group(fas_group):
         db.session.add(group)
         db.session.commit()
 
-        flask.flash(
-            "FAS group {} is activated in the Copr under the alias {} "
-            .format(fas_group, alias)
-        )
+        # can't have more than one alias for a group
+        message = "Group {} is activated in the System under the alias {} ".format(
+            fas_group, alias)
+        flask.flash(message, "success")
         return flask.redirect(url_for(
-            "groups_ns.list_projects_by_group", group_name=alias))
+            "groups_ns.list_projects_by_group", group_name=group.name))
```

Comment:
This is certainly OK and better behavior than before, thank you!

What I thought meant before was that we should redirect the user (or error 400) much earlier, before the user starts even thinking about how to name the group (it is wasting of time, the group is already activated so nothing will happen).  Something like:

```python
def activate_group(fas_group):
    if "fas group is activated":
        return flask.redirect(...)
```

Reply:
That's sounds a good idea, will tweak the code :)

Reply:
> This is certainly OK and better behavior than before, thank you!
> 
> What I thought meant before was that we should redirect the user (or error 400) much earlier, before the user starts even thinking about how to name the group (it is wasting of time, the group is already activated so nothing will happen). Something like:
> 
> ```python
> def activate_group(fas_group):
>     if "fas group is activated":
>         return flask.redirect(...)
> ```

Done